homeassistant/components/huisbaasje/const.py

```diff
@@ -0,0 +1,86 @@
+"""Constants for the Huisbaasje integration."""
+from huisbaasje.const import (
+    SOURCE_TYPE_ELECTRICITY,
+    SOURCE_TYPE_ELECTRICITY_IN,
+    SOURCE_TYPE_ELECTRICITY_IN_LOW,
+    SOURCE_TYPE_ELECTRICITY_OUT,
+    SOURCE_TYPE_ELECTRICITY_OUT_LOW,
+    SOURCE_TYPE_GAS,
+)
+
+from homeassistant.const import (
+    DEVICE_CLASS_POWER,
+    ENERGY_KILO_WATT_HOUR,
+    TIME_HOURS,
+    VOLUME_CUBIC_METERS,
+)
+
+DOMAIN = "huisbaasje"
+
+FLOW_CUBIC_METERS_PER_HOUR = f"{VOLUME_CUBIC_METERS}/{TIME_HOURS}"
+
+POLLING_INTERVAL = 10
+"""Interval in seconds between polls to huisbaasje."""
```

Comment:
Comments in Python should be like this:

```suggestion
# Interval in seconds between polls to huisbaasje.
POLLING_INTERVAL = 10
```

The other format is just for doc strings at the start of Python modules, classes and functions.

Reply:
Hmm. Very consistent :(.